src/main/java/org/opentripplanner/routing/graph/Graph.java

```diff
@@ -119,8 +120,8 @@
 
     private GraphBundle bundle;
 
-    /* Vertex index by name is reconstructed from edges. TODO Is this actually needed? */
-    private transient Map<String, Vertex> vertices;
+    /* Ideally we could just get rid of vertex labels, but they're used in tests and graph building. */
+    private Map<String, Vertex> vertices = new ConcurrentHashMap<>();
 
```

Comment:
This is just a difference in mathematical and software vocabulary, as in https://en.wikipedia.org/wiki/Graph_labeling. We could rename them, but I think this would rename a lot of field references in tests etc. and cause additional drift away from OTP1, potentially complicating merges in the future.

Reply:
Thank you for clarifying that - I did not make the connection to the mathematical - lets keep label an add a comment explaining why.

Reply:
Added a comment on `label` in Vertex.